### PR TITLE
Fix vectorize function depwarn on 0.6

### PR DESCRIPTION
**Pull request by yuyichao**
Requires https://github.com/JuliaLang/Compat.jl/pull/280 and a new `Compat` tag.

### CODE REVIEW

File: REQUIRE [hunk 1]
```diff
@@ -1,2 +1,2 @@
 julia 0.4
-Compat 0.7.14
+Compat 0.8.9
```
Comment (tkelman) [inline REQUIRE]: this won't be right

Comment (yuyichao) [inline REQUIRE]: 0.8.9 wasn't there when I made the change 8 hours ago.

Comment (tkelman) [inline REQUIRE]: it's not going to be 0.8.10 either

Comment (yuyichao) [inline REQUIRE]: Well, I don't really care what it is going to be right now. So if I can't put a place holder like `0.8.8+` and I can't just put what is currently the next version should I just leave the version requirement unchanged?

Comment (tkelman) [inline REQUIRE]: no, better to hold off on opening the pr if it shouldn't be merged yet

**Comment by yuyichao** [comment]
Compat.jl PR merged and tagged. Version requirements updated and CI passed. Should be ready to go.

Also updated travis to also run on 0.5.

**Pull request merged**